test/metabase/test/data/sql.clj

```diff
@@ -348,6 +348,18 @@
            query           (str/replace query (re-pattern #"(.*)(?:1337)(.*)") (format "$1%s$2" to-insert))]
        {:query query}))))
 
+(defmethod tx/make-alias :sql/test-extensions
+  ([_driver alias]
+   alias))
+
+(defmethod tx/make-alias :oracle
+  ([_driver alias]
+   (str "\"" alias "\"")))
+
+(defmethod tx/make-alias :snowflake
+  ([_driver alias]
+   (str "\"" alias "\"")))
```

Comment:
You shouldn't put driver-specific implementations in a general namespace like this